includes/class-migration.php

```diff
@@ -955,11 +955,17 @@ public static function update_notification_options() {
 		\add_option( 'activitypub_blog_user_mailer_new_follower', $new_follower );
 		\add_option( 'activitypub_blog_user_mailer_new_mention', '1' );
 
+		$users = \get_users(
+			array(
+				'capability__in' => array( 'activitypub' ),
```

Comment:
```suggestion
				'capability__in' => array( 'activitypub' ),
				'fields'         => 'id',
```
If we only need IDs, this should work, too